ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/layout.tsx

```diff
@@ -1,26 +1,6 @@
-'use client';
-
-import { useCallback } from 'react';
-import { Badge } from '@inngest/components/Badge';
-import { InvokeButton } from '@inngest/components/InvokeButton';
-import { IconFunction } from '@inngest/components/icons/Function';
-import { useMutation } from 'urql';
-
-import { ArchivedAppBanner } from '@/components/ArchivedAppBanner';
-import { useEnvironment } from '@/components/Environments/environment-context';
-import { useBooleanFlag } from '@/components/FeatureFlags/hooks';
-import Header, { type HeaderLink } from '@/components/Header/old/Header';
-import { graphql } from '@/gql';
-import { useFunction } from '@/queries';
-import { pathCreator } from '@/utils/urls';
-import { CancelFunctionButton } from './CancelFunctionButton';
-import PauseFunctionButton from './PauseButton';
-
-const InvokeFunctionDocument = graphql(`
-  mutation InvokeFunction($envID: UUID!, $data: Map, $functionSlug: String!) {
-    invokeFunction(envID: $envID, data: $data, functionSlug: $functionSlug)
-  }
-`);
+import { getBooleanFlag } from '@/components/FeatureFlags/ServerFeatureFlag';
```

Comment:
Again, addd an ssr layout above the new and old client side layouts so we don't flicker in the old designs before the flag is fetched.